x-pack/plugins/cases/public/types.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CoreStart } from 'kibana/public';
+import {
+  TriggersAndActionsUIPublicPluginSetup as TriggersActionsSetup,
+  TriggersAndActionsUIPublicPluginStart as TriggersActionsStart,
+} from '../../triggers_actions_ui/public';
+
+export interface SetupPlugins {
+  triggersActionsUi: TriggersActionsSetup;
+}
+
+export interface StartPlugins {
+  triggersActionsUi: TriggersActionsStart;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface PluginStart {}
```

Comment:
As we expose `getCreateCase` and `casesComponent` we should fill this interface accordingly.

Reply:
i deleted it, i had a different type defining this `CasesUiStart`